src/app/share/views/ShareView/ShareFileView.tsx

```diff
@@ -270,7 +270,7 @@ export default function ShareFileView(props: ShareViewProps): JSX.Element {
               <abbr className="w-screen max-w-prose break-words px-10 text-xl sm:w-full" title={getFormatFileName()}>
                 {getFormatFileName()}
               </abbr>
-              <span className="text-cool-gray-60">{getFormatFileSize()}</span>
+              <span className="text-cool-gray-60">{getFormatFileSize()}lala</span>
```

Comment:
Remove "lala"

Reply:
I mark it as solved because I saw it in another [task](https://github.com/internxt/drive-web/pull/732) that has to be merged and it is already removed.